src/scss/components/_serviceRichText.scss

```diff
@@ -0,0 +1,40 @@
+.richtext_title {
+  @include heading-4;
+
+  color: var(--brand-navy);
+  margin: 0;
+  padding-top: 6.4rem;
+}
+
+.richtext_container {
+  > h3 {
+    color: var(--brand-navy);
+    @include heading-5;
+
+    margin: 0;
+    padding-top: 1.6rem;
+  }
+
+  > p {
+    @include body-m-reg;
+
+    color: var(--grey-2);
+    margin: 0;
+    padding-top: 0.8rem;
+  }
+
+  > ul {
+    margin: 0;
+    padding-top: 0.8rem;
+
+    > li {
+      > p {
+        color: var(--grey-2);
+        @include body-m-reg;
+
+        margin: 0;
+        list-style-type: disc;
```

Comment:
Here the list-style-type:disc is not being rendered in the page. This attribute should be part of the <ul> tag as per documentation.

Reply:
Hi Alex u are Right, ill change that, ty! :)